clarity/src/vm/database/clarity_db.rs

```diff
@@ -880,11 +887,12 @@ impl<'a> ClarityDatabase<'a> {
 
     /// Returns the _current_ total liquid ustx
     pub fn get_total_liquid_ustx(&mut self) -> Result<u128> {
+        let epoch = self.get_clarity_epoch_version()?;
         Ok(self
             .get_value(
                 ClarityDatabase::ustx_liquid_supply_key(),
                 &TypeSignature::UIntType,
-                &StacksEpochId::latest(),
```

Comment:
I see what you're doing here, but I think we'll want to consider a better way to do this.  We're using `self.get_clarity_epoch_version()` because this will return `StacksEpochId::Epoch32` until the hard fork which activates Clarity 4 -- at which point, it will return `StacksEpochId::Epoch33`, right?  If so, then if there's ever a hard fork which bumps the epoch without bumping Clarity, then the Clarity 4 epoch (`Epoch33`) would continue to be passed in here instead of the latest epoch, right?